cmd/explaintest/config.toml

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 lease = "0"
-mem-quota-query = 34359738368
```

Comment:
It looks like removing this line is causing the explaintest to fail. It might require 32G memory vs. the default of 1G to run.

I checked out your branch locally, and it looks like this fixes it:

```
diff --git a/cmd/explaintest/main.go b/cmd/explaintest/main.go
index 71c5ef17a..9e286a633 100644
--- a/cmd/explaintest/main.go
+++ b/cmd/explaintest/main.go
@@ -721,6 +721,8 @@ func main() {
                "set @@tidb_projection_concurrency=4",
                "set @@tidb_distsql_scan_concurrency=15",
                "set @@global.tidb_enable_clustered_index=0;",
+               "set @@global.tidb_mem_quota_query=34359738368",
+               "set @@tidb_mem_quota_query=34359738368",
        }
        for _, sql := range resets {
                if _, err = mdb.Exec(sql); err != nil {
```

Reply:
Thanks. I canceled the modification.

Reply:
Thanks!! :slightly_smiling_face: